test_settings.py

```diff
@@ -16,3 +16,11 @@
         'cms.test_utils.project.sampleapp'
     ],
 }
+
+
+def run():
```

Comment:
Why this and not just invoking the djangocms executable? The reason I'm asking is not because I like one better than the other but because I want to learn what is the canonical way to run addon tests.